real-main/app/clients/id_verification.py

```
@@ -56,3 +61,20 @@ def verify_id(self, user_id, frontside_image, country, id_type, mime_type):
             raise Exception(
                 f'Unable to parse response from jumio verification service with body: `{resp.text}`'
             ) from err
+
+    def verify_id_with_id_analyzer(self, frontside_image):
+        if not self.id_analyzer_api_key:
+            return
```

I think this check could hide errors if the entries in the SecretsManager are misconfigured? Better to fail early and loudly....

If you're looking for an easy way to disable the service, I would do that either with another property in the secrets manager json or possibly with a special api key of something like `DISABLED`